### ISSUE TITLE
Try/catch variable not detected

### ISSUE THREAD

**Issue by chrisdp** [rokucommunity/bslint#48]
```vb
		try
			result = 0 / 0
			print result
		catch e
			' temporary debugging code
			throw e
		end try
```

Currenly `e` is being detected as an uninitialized variable on line with `throw e`
